src/templates/view-edit-toolbar.html

```
@@ -155,190 +155,201 @@ <h5 class='modal-title' id='procurementRequestModalLabel'>{{ 'Create procurement
+      </a>
+    {% endif %}
+
+    <div class='vertical-separator'></div>
+
+    {% if mode == 'edit' %}
+      {# VIEW button #}
+        <button type='button' title='{{ 'Save'|trans }}' data-action='update-entity-body' aria-label='{{ 'Save'|trans }}' class='btn btn-primary mr-2'>
+          <i class='fas fa-save'></i>
         </button>
-      {% endif %}
+      <a title='{{ 'Save and go back'|trans }}' aria-label='{{ 'Save and go back'|trans }}' href='?mode=view&amp;id={{ Entity.id }}' data-redirect='view' class='btn btn-transparent mr-2'>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)